packages/firebase_remote_config/android/src/main/java/io/flutter/plugins/firebase/firebaseremoteconfig/MethodCallHandlerImpl.java

```diff
@@ -124,7 +124,7 @@ public void onComplete(Task<Boolean> task) {
       case "RemoteConfig#setDefaults":
         {
           Map<String, Object> defaults = call.argument("defaults");
-          FirebaseRemoteConfig.getInstance().setDefaults(defaults);
```

Comment:
`setDefaultsAsync` is an asynchronous method which returns a Task. We might need to add a completion listener to it, same way it's done on `RemoteConfig#activate` case above.

Reply:
I hope fix for deprecated warning.